desktop_native/src/password/unix.rs

```diff
@@ -0,0 +1,83 @@
+use anyhow::{anyhow, Result};
```

Comment:
In general, all macos comments apply here as well.
* why async?
* why specified lifetimes?
* prefer `?` over `map_error`
* test needs work to not orphan secrets on failure